app/javascript/src/components/Dashboard/Notes/NoteList/Note.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { Clock } from "@bigbinary/neeto-icons";
+import { Avatar, Tooltip } from "@bigbinary/neetoui";
+
+import DropDownNote from "./DropDownNote";
+
+const Note = ({
+  note,
+  setSelectedDeleteNoteIds,
+  setSelectedNote,
+  setShowEditNote,
+  setShowDeleteAlert,
+}) => {
+  const handleEditClick = () => {
+    setSelectedNote(note);
+    setShowEditNote(true);
+  };
+  const handleDeleteClick = () => {
+    setSelectedDeleteNoteIds([note.id]);
+    setShowDeleteAlert(true);
+  };
+  return (
+    <div className="notes-font order-none mb-4 box-border flex h-40 flex-none flex-grow-0 flex-col items-start self-stretch rounded-sm border border-solid border-[#d8dcde] bg-white p-4 shadow-[0_1px_4px_-1px_rgba(28,48,74,0.12)]">
```

Comment:
No need to update the colors via Tailwind classes. We don't allow doing that in neeto codebases. Did you add these colors because it was mentioned in the Figma design?

Reply:
Yes, those colours were mentioned in the Figma design. I will add those specs in the stylesheet rather than updating them via tailwind classes. Thanks for informing Yedhin.

Reply:
Actually what I was trying to say is that, we don't need to add any new custom colors other than the ones mentioned at https://neetoui.onrender.com/?path=/docs/foundation-colors--page. Because if we start choosing custom colors then it will deviate from the design pattern followed by neetoUI.

I will update the Figma designs later and remove unnecessary colors.

Reply:
Oh I see. Thanks Yedhin.